docs/api/browser-window.md

```diff
@@ -169,6 +169,15 @@ Emitted when window is minimized.
 
 Emitted when window is restored from minimized state.
 
+### Event: 'resize'
+
+Emitted when window is getting resized on all platforms.
+
+### Event: 'move'
```

Comment:
Any chance this could be made the same on all platforms? Every little platform difference makes it harder to create apps.